pkgs/top-level/python-packages.nix

```diff
@@ -8522,7 +8526,13 @@ in {
 
   webthing = callPackage ../development/python-modules/webthing { };
 
-  werkzeug = callPackage ../development/python-modules/werkzeug { };
+  werkzeug = callPackage ../development/python-modules/werkzeug {
+    pytestCheckHook = self.pytestCheckHook.override { pytest = self.pytest_6_1; };
+    pytest-timeout = self.pytest-timeout.override {
+      pytest = self.pytest_6_1;
```

Comment:
pytest plugins/extensions should not propagate pytest, see c44ae94eab1bcf4dbef5405800fc597d0b4f78dd